### PR TITLE
Included the subscribed channel list also.

### PR DESCRIPTION
**Pull request by ushukkla**
I have included the channel link, to which any user is being subscribed. Through this, he/she may directly add the video in his/her channel to the playlist, without browsing.

### CODE REVIEW

**Comment by ushukkla** [comment]
Included the CSS File (inject.css) in inject.js

File: src/inject/inject.js [hunk 1]
```diff
@@ -40,6 +44,10 @@ function insertButton() {
         to_match = to_match.concat("yt-uix-tile-link yt-ui-ellipsis yt-ui-ellipsis-2 yt-uix-sessionlink      spf-link ", '"]');
         LOC_HREF = 4;
     }
+    else if (document.location.href == "https://www.youtube.com/feed/channels") {
+        to_match = to_match.concat("yt-uix-tile-link yt-ui-ellipsis yt-ui-ellipsis-2 yt-uix-sessionlink      spf-link ", '"]');
+        LOC_HREF = 5;
```
Comment (nishnik) [inline src/inject/inject.js:61]: I don't think this works. Have you checked it?

IMO there should be substring check like on line no 35.
https://github.com/Manu-shukla/Play_Next/blob/c4baf927cc5a065c8be96866085ae160f3612b52/src/inject/inject.js#L35

Comment (ushukkla) [reply src/inject/inject.js:61]: But, locally it is working on my system. I think it is due to cookie being saved, I'll check it once.

Comment (nishnik) [reply src/inject/inject.js:61]: Could you post a screenshot if it works on your laptop, even if it works for your personal channel IMO it won't work for every channel. Kindly check

Comment (ushukkla) [reply src/inject/inject.js:61]: Ok, let me check.

**Pull request closed without merging**